### ISSUE TITLE
VLC won't play stream already in progress

### ISSUE THREAD

**Issue by amki** [sergey-dryabzhinsky/nginx-rtmp-module#105]
VLC (v2.2.2) can't open streams already running when it connects.
If VLC is being connected before publishing starts everything works perfectly.
I'm using the following config:
```
rtmp {
    server {
        listen 1935;
        listen [::]:1935 ipv6only=on;
        chunk_size 1024;

        application vecors {
            live on;
            meta on;
            wait_key on;
            publish_notify on;
            play_restart on;

            allow publish all;
            allow play all;
        }
    }
}
```
Is that an error on my side or is this a bug?

**Comment by farfelu** [comment]
I'm having the same issues. Clients can only connect before publishing starts and a short timespan after that.
I did some testing in the discussion https://github.com/sergey-dryabzhinsky/nginx-rtmp-module/issues/54#issuecomment-189753044 but couldn't figure it out what was causing it.

**Comment by amki** [comment]
This happens if you have more than one nginx worker. Using only one worker (worker_processes  1;
) works for me.

**Comment by chriswiggins** [comment]
Yeah more than one worker is broken at the moment. We really should fix it :-)

On 29/04/2016, at 7:08 AM, amki <notifications@github.com<mailto:notifications@github.com>> wrote:


This happens if you have more than one nginx worker. Using only one worker (worker_processes 1;
) works for me.

-

**Comment by misiek08** [comment]
It's not easy to fix it. Communicating between processes isn't the easiest
thing one can implement and in this case to maintain this module
performance it would require to do it very slowly and precisely.

**Comment by ReubenM** [comment]
Not sure how useful it would be since I'm not very familiar with nginx internals, but there are existing worker IPC implementations that could be useful to take a look at.

https://github.com/wandenberg/nginx-push-stream-module/blob/master/src/ngx_http_push_stream_module_ipc.c
